runtime/serde/serde-xml/common/src/aws/smithy/kotlin/runtime/serde/xml/XmlDeserializer.kt

```diff
@@ -5,7 +5,6 @@
 
 package aws.smithy.kotlin.runtime.serde.xml
 
-import aws.smithy.kotlin.runtime.logging.Logger
```

Comment:
question
if we don't use it at all can we drop the dependency?

Reply:
Yes, sure can. I wish we had a ktlint-style rule for removing unused dependencies like we have for unused imports.